tests/transactions/e2e/features/send-money-to-receiver.feature

```diff
@@ -3,12 +3,13 @@ Feature: Send Money to Receiver
 
   Background:
     Given I have completed the basic setup
-    And I have the following "Rewards" wallets:
+    And I have the following wallets:
       | name   |
       | first  |
 
+  @api-wip-byron
```

Comment:
@tomothespian this test should be working as the "Send" screen is fully functional on Byron builds as well as on ITN builds.